realm/realm-annotations-processor/src/main/java/io/realm/processor/Constants.java

```diff
@@ -71,4 +71,26 @@
         JAVA_TO_COLUMN_TYPES.put("java.util.Date", "RealmFieldType.DATE");
         JAVA_TO_COLUMN_TYPES.put("byte[]", "RealmFieldType.BINARY");
     }
+
+    static final Map<String, String> JAVA_TO_FIELD_SETTER;
```

Comment:
The rest of the constants are about mapping from from Java -> Realm, in this particular case it is about mapping from Java -> Java, but it is just a minor thing.